packages/manager/modules/nutanix/src/dashboard/nodes/node/general-info/routing.js

```diff
@@ -8,11 +8,25 @@ export default /* @ngInject */ ($stateProvider) => {
     },
     resolve: {
       nodeId: /* @ngInject */ ($transition$) => $transition$.params().nodeId,
-      orderPrivateBandwidthLink: /* @ngInject */ ($state, nodeId) =>
+      handleError: /* @ngInject */ (Alerter) => (error) => {
+        Alerter.alertFromSWS(error.message, 'ERROR', 'nutanix_node_alert');
+      },
```

Comment:
Do you really use that function ? Seems not

Reply:
Yes it is passed to the component as "onError" parameter. This was previous behavior which is why it's not visible on this PR but you can see it in private-order.controller :

    if (isFunction(this.onError)) {
      this.onError({
        error: { message, data: error },
      });
    }